converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/RetryableFhirTransactionPipeline.cs

```diff
@@ -49,14 +35,19 @@ public RetryableFhirTransactionPipeline(IFhirTransactionPipeline fhirTransaction
                 .Handle<RetryableException>()
                 .WaitAndRetryAsync(
                     maxRetryCount,
-                    retryAttempt => TimeSpan.FromSeconds(Math.Pow(2, retryAttempt)),
+                    (retryAttempt) =>
+                    {
+                        double waitTime = Math.Pow(2, retryAttempt);
```

Comment:
use min(60, math.pow(2, retryAttempt)) to get rid of ternary operator

Reply:
Updated